script/Common/ORPHANED_LINES.txt

```diff
@@ -1,27 +1,30 @@
 [sha:6a4ad8158d13c1c44b6bda2ba0c5075d72a55da7]{
 -- Page -1, Offset 12.
 -- もし、空が雲っていたのなら。
--- TRANSLATION HERE
+// This whole file is one 'page' so the quote checker gets confused.
+// lint-off:LintUnclosedQuotes
+What if, the sky had been cloudy.
 }
 [sha:894329ed51deb533fd3c82fed12c93b297574eea]{
 -- Page -1, Offset 13.
 -- もし、気づくのが少し早ければ。
--- TRANSLATION HERE
+What if, I had noticed a little sooner.
 }
 [sha:6f86bd9c66e5fbf76bb8fe4e650b4ebc23bd9cc5]{
 -- Page -1, Offset 14.
 -- もし、彼がおとろえていなければ。
--- TRANSLATION HERE
+// 衰える?
+What if, he hadn't weakened.
```

Comment:
https://forums.nrvnqsr.com/showthread.php/8981-SUMMARY-TRANSLATIONS-Tsukihime-A-piece-of-blue-glass-moon-%28CONTAINS-SPOILERS%29?p=3149175&viewfull=1#post3149175

Suppose I can shill my own translation of this scene.